### PR TITLE
Feat hide no video participants

### PR DESCRIPTION
**Pull request by Astagor**
I have added option of hiding of peers without a video nor presentation. Should be a solution for #667

### CODE REVIEW

File: app/src/translations/cs.json [hunk 1]
```diff
@@ -171,6 +171,7 @@
 	"settings.noiseThreshold": "Mezní hodnota hluku",
 	"settings.mirrorOwnVideo": "Zrcadlové zobrazení vlastního videa",
 	"settings.language": "Vyberte jazyk",
+	"settings.hideNoVideoParticipants": null,
```
Comment (jmencak) [inline app/src/translations/cs.json:174]: Czech translation: 
`"settings.hideNoVideoParticipants": "Zobrazovat jen účastníky s obrazem",`

**Review by jmencak — APPROVED** [review]
Thank you for the PR.  I've tested it and it works fine.  Feel free to add Czech translation for this feature I provided in the review.

**Comment by Astagor** [comment]
@so010 I have change the way it works, no watching of store.

**Review by so010 — APPROVED** [review]
Looks god for now but we have to restructure spotlights.

**Pull request merged**